main.go

```diff
@@ -10,13 +10,46 @@ import (
 	"github.com/opencontrol/fedramp-templater/templater"
 )
 
-func parseArgs() (openControlsDir, inputPath, outputPath string) {
-	if len(os.Args) != 4 {
-		log.Fatal("Usage:\n\n\tfedramp-templater <openControlsDir> <inputDoc> <outputDoc>\n\n")
+type options struct {
+	openControlsDir string
+	inputPath string
+	outputPath string
+	diff bool
+	fill bool
+}
```

Comment:
This is admittedly nitpicky, but since `diff` and `fill` are mutually exclusive, how about representing them as

```go
type subCommand int

const (
        diff subCommand = iota
        fill
)

type options struct {
        // ...
        cmd subCommand
}